src/node.js

```diff
@@ -340,7 +340,8 @@
     function _tickCallback() {
       var callback, threw, tock;
 
-      scheduleMicrotasks();
+      if (microtasksScheduled)
```

Comment:
ah, wow. totally missed what you were saying about the conditional. let me fix it and push up a fix. one moment.